cognite/client/_api/data_modeling/instances.py

```diff
@@ -1282,7 +1287,11 @@ def aggregate(
 
         self._validate_filter(filter)
         filter = self._merge_space_into_filter(instance_type, space, filter)
-        body: dict[str, Any] = {"view": view.dump(camel_case=True), "instanceType": instance_type, "limit": limit}
+        body: dict[str, Any] = {
+            "view": view.dump(camel_case=True),
+            "instanceType": instance_type,
+            "limit": self._AGGREGATE_LIMIT if is_unlimited(limit) or limit is None else limit,
```

Comment:
```suggestion
            "limit": self._AGGREGATE_LIMIT if is_unlimited(limit) else limit,
```

ref its implementation:

```py
def is_unlimited(limit: float | int | None) -> bool:
    return limit in {None, -1, math.inf}
```

Reply:
The `limit is None` is just for `mypy`.

Reply:
My `mypy` is happy without it

Reply:
Ok then it is me that did not trust mypy